js/src/BarsModel.ts

```diff
@@ -103,6 +103,7 @@ export class BarsModel extends markmodel.MarkModel {
                     if (isNaN(value)) {
                         value = 0;
                     }
+
```

Comment:
Can we remove this from the commit since there are no longer any changes in this file?